docs/guides/map-use-case-to-provider.md

```diff
@@ -85,9 +120,9 @@ map GetWeather {
 
 ### Make HTTP request {#http-request}
 
-Define the first request by specifying the `http` block with HTTP method and URL path.
+Prepared map contains numbered comments guiding you through mapping the use case. Start with specifying the HTTP method and URL path. Provider's default service base URL is provided in comment for your convinience. See [Comlink reference](https://superface.ai/docs/comlink/map#sec-HTTP-Call) for specifying a different service.
 
-All widely known HTTP methods are supported. The path is a simple _string_, but can be templated. _E.g. if you need to request resource whose ID was provided in the input, you might want to use something like `/resources/{input.id}`._
+All widely known HTTP methods are supported. The path is a simple _string_, but can be templated. _E.g. if you need to request resource whose ID was provided in the input, you might want to use something like `/endpoint/api/{ input.field }`._
```

Comment:
```suggestion
All widely known HTTP methods are supported. The path is a simple _string_, but can be templated. _E.g. if you need to request a resource whose ID was provided in the input, you might want to use something like `/endpoint/api/{ input.field }`._
```

Actually, I think I prefer `/resources/{input.id}` - why change it to `/endpoint/api`?

Reply:
`/endpoint/api` is used in map template. I was thinking that examples in text could use code from templates to give user more context.

Reply:
Oh! Ok